README.md

```diff
@@ -173,6 +173,37 @@ const styles = StyleSheet.create({
 
 Aphrodite will ensure that the global `@font-face` rule for this font is only inserted once, no matter how many times it's referenced.
 
+# Descendant selectors
+
+Descendant rules (i.e. a rule like `.parent:hover .child { ... }` in CSS) are created by adding values to your styles indicating the styles to apply to the child. To distinguish the styles from normal CSS keys, the keys start with a `>>`, like `'>>child'`.
```

Comment:
On initial read of this, I assumed that `child` would be a literal class name and it took me a couple read throughs to figure out that `child` would be a classname generated by Aphrodite.

It might be worth explaining here that this is really only for cases that can't be expressed directly through others mean (which, I believe, is only styling children of pseudo-selector-modified parents).